src/yunohost/settings.py

```diff
@@ -74,6 +74,7 @@ def is_boolean(value):
     ("smtp.relay.user", {"type": "string", "default": ""}),
     ("smtp.relay.password", {"type": "string", "default": ""}),
     ("backup.compress_tar_archives", {"type": "bool", "default": False}),
+    ("panel_overlay.enabled", {"type": "bool", "default": True}),
```

Comment:
I guess we could have more option around ssowat (like themas...)

I suggest to call that : ssowat.panel_overlay.enabled or something like that

Reply:
Indeed that would make sense

Reply:
Done !